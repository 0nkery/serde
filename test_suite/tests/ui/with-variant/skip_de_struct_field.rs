#[macro_use]
extern crate serde_derive;

#[derive(Deserialize)]
enum Enum {
    #[serde(deserialize_with = "deserialize_some_other_variant")]
    Struct {
        #[serde(skip_deserializing)]
        f1: String,
        f2: u8,
    },
}

fn main() {}
