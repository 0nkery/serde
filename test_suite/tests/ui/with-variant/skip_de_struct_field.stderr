error: variant `Struct` cannot have both #[serde(deserialize_with)] and a field `f1` marked with #[serde(skip_deserializing)]
 --> $DIR/skip_de_struct_field.rs:4:10
  |
4 | #[derive(Deserialize)]
  |          ^^^^^^^^^^^

error: aborting due to previous error

